### ISSUE TITLE
explorer: Show cluster staking information

### ISSUE THREAD

**Issue by jstarry** [solana-labs/explorer#218]
#### Problem
- Want to see list of validators, map, and live leader schedule
- Want to see stake distribution, delinquency, skipped blocks %, and staking instructions

#### Proposed Solution
- Add new `/staking` route
- Use Solana Beach UI components
    - Staking card (delegated, warmed up, active, delinquent)
    - Live leader schedule + map card
    - Validator list table for stake distribution

**Comment by rogaldh** [comment]
@Woody4618 Do you think we need this info? I think this is solved already at https://validators.app
I think some information could be added, but I'm not sure if that would help.

**Comment by Woody4618** [comment]
Agreed we have the link to validators.app where ppl can dig into more info
